src/native-common/Button.tsx

```diff
@@ -82,16 +82,17 @@ export class Button extends ButtonBase {
     touchableGetInitialState!: () => RN.Touchable.State;
     touchableHandleStartShouldSetResponder!: () => boolean;
     touchableHandleResponderTerminationRequest!: () => boolean;
-    touchableHandleResponderGrant!: (e: React.SyntheticEvent<any>) => void;
-    touchableHandleResponderMove!: (e: React.SyntheticEvent<any>) => void;
-    touchableHandleResponderRelease!: (e: React.SyntheticEvent<any>) => void;
-    touchableHandleResponderTerminate!: (e: React.SyntheticEvent<any>) => void;
+    touchableHandleResponderGrant!: (e: RN.GestureResponderEvent) => void;
+    touchableHandleResponderMove!: (e: RN.GestureResponderEvent) => void;
+    touchableHandleResponderRelease!: (e: RN.GestureResponderEvent) => void;
+    touchableHandleResponderTerminate!: (e: RN.GestureResponderEvent) => void;
 
-    private _isMounted = false;
+    protected _isMounted = false;
     protected _isMouseOver = false;
     protected _isHoverStarted = false;
+    protected _buttonElement: any = null;
```

Comment:
Similar to before (and throughout), shouldn't this be RN.Animated.View?

Reply:
Unfortunately, I can't type it as RN.Animated.View because this isn't a class definition, only a type definition (i.e. it can be used as a JSX tag but not as a variable type).